test/eth.accounts.signTransaction.js

```diff
@@ -652,29 +652,29 @@ var tests = [
         transactionHash: "0xbc2c9edab3d4e3a795fa402b52d6149e874de15f0cc6c0858eb34e1fe1ef31fe",
         messageHash: "0xa3a2cdc45e9cefb9a614ead90ce65f68bcf8a90dbe0ccbd84c1b62403bd05346"
     },
-    {
-        // test #28
-        address: '0x2c7536E3605D9C16a7a3D7b1898e529396a65c23',
-        iban: 'XE0556YCRTEZ9JALZBSCXOK4UJ5F3HN03DV',
-        privateKey: '0x4c0883a69102937d6231471b5dbb6204fe5129617082792ae468d01a3f362318',
-        transaction: {
-            chainId: 1,
-            nonce: 0,
-            maxPriorityFeePerGas: new bn('0x3B9ACA00'),
-            maxFeePerGas: new bn('0xB2D05E00'),
-            gasLimit: '0x6A40',
-            to: '0xF0109fC8DF283027b6285cc889F5aA624EaC1F55',
-            toIban: 'XE04S1IRT2PR8A8422TPBL9SR6U0HODDCUT', // will be switched to "to" in the test
-            value: "1000000000",
-            data: "",
-            common: commonLondon
-        },
-        // signature from eth_signTransaction
-        rawTransaction: "0x02f86e018084c733124884cb72ec20826a4094f0109fc8df283027b6285cc889f5aa624eac1f55843b9aca0080c001a08896fb9a5c033e0163b073cf7a951a1db2dca41b26b4188f13a05158eb26fd32a005e8855691199cd0b6dcae88f3325c374e2f0697b9c528a5c10d5bd8dfb6a3e3",
-        oldSignature: "0x02f86e018084c733124884cb72ec20826a4094f0109fc8df283027b6285cc889f5aa624eac1f55843b9aca0080c001a08896fb9a5c033e0163b073cf7a951a1db2dca41b26b4188f13a05158eb26fd32a005e8855691199cd0b6dcae88f3325c374e2f0697b9c528a5c10d5bd8dfb6a3e3",
-        transactionHash: "0xd5b7290a477b9c421d39e61d0f566ec33276fb49b9ff85cfd6152a18f1c92dab",
-        messageHash: "0x17e20e530a889ce52057de228b5b97edcad6002468d723346cd0b6b7a9943457"
-    },
+    // {
```

Comment:
why the test is commented on?

Reply:
This testcase is not correct, we were using bn version 4.11.9 and an error was [found](https://github.com/indutny/bn.js/issues/151) parsing hexstirngs incorrectly. it has been incorrectly converting hexstrings to decimals. 
```
> var a = new web3.utils.BN("0x3B9ACA00", 16)
undefined
> a
BN { negative: 0, words: [ 60475904, 526 ], length: 2, red: null }
> 
```
But `0x3B9ACA00` should be `1000000000`

So with the new bn.js update for our dependancies, there was a [breaking change](https://github.com/indutny/bn.js/releases/tag/v5.0.0) and now this testcase is failing and i had to comment it out. Will need to investigate this

Reply:
I believe it's a very important test in `signTransaction` so we should not disable it, but rather investigate it and also include its fix in the same release. 

If the upgrade of `bn.js` is causing some misbehavior to transaction signature then we should report the issue to them and don't upgrade `bn.js` right away.

Reply:
From the tests I have run, the misbehavior seems to be only in the 4.11.9 version, so I just updated the hex values in the test and reenabled it.

I noticed that ethereumjs-utils uses bn:4.11.6, maybe, we should let them know

Reply:
test is uncommented and works fine now with @nikoulai's [fix](https://github.com/ChainSafe/web3.js/pull/5074/files#diff-bd97e9c608060469030c1d6a655baa533c0aa0b11939b0670ebdb41aa51d786bR663), the issue is `bn` errors when given strings starting with`0x`, we shouldn't have passed ` bn('0x3B9ACA00')` but instead it should have been `bn('cb72ec20', 16),`